spec/views/view_helpers_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "spec_helper"
+
+describe "Clearance RSpec view spec configuration" do
```

Comment:
Also, do you want a `type: :helper` spec to test that it's included in helper specs too?